docs/java/release-notes.md

```diff
@@ -27,13 +27,73 @@ We highly recommend regularly updating to the latest SDK version. It will help y
 - update client libraries giving access to latest SAP services on SAP Cloud Platform and S4/HANA
 - protect yourself from bugs and breaking changes in the future
 
+## 3.26.0
+
+- [JavaDoc](https://help.sap.com/doc/170aace62830463d8f4ec9a9251667ee/1.0/en-US/index.html)
+
+### New functionality
+
+#### OData and type-safe client libraries
+
+- We released a Beta version of improved type-safe client library implementation for OData v2 protocol which now shares a code base with OData v4 protocol. It brings significant advantages like:
+  - Boosting performance by avoiding unnecessary metadata calls
+  - The same code base for both OData protocol versions speeds up the innovation cycle and unlocks the developing of advanced features for both OData versions.
+  - For more insights and guide how to switch to an new version check our [type-safe client documentation](features/odata/use-typed-odata-v2-and-v4-client-for-java#switch-to-improved-odata-vdm-beta)
+- The results of `create` and `update` operations in OData v4 type-safe client now offer `getModifiedEntity()` method which returns a new `Entity object` including any changes to payload or metadata the service may have responded with.
+- We released a regular update for pre-generated type-safe client libraries (also known as VDM) for the [latest RTC release 2008 of SAP S/4HANA Cloud](https://news.sap.com/2020/08/sap-s4hana-cloud-release-2008-stay-ahead-change/).
```

Comment:
I suggest to split this visibly from the points about the new beta client above (new heading or indent all lines except first one for beta client)

Reply:
I'll merge this and implement the improvements you've requested, @sacnl